### PR TITLE
Put all formatted output command line args into a single group

### PR DESCRIPTION
**Pull request by clumens**
Also, rename them from --output-whatever to --<format>-whatever.

### CODE REVIEW

File: doc/Pacemaker_Administration/en-US/Ch-Tools.txt [hunk 1]
```diff
@@ -16,19 +16,21 @@ command line options for changing the style of the output.  For instance:
...
-  --output-meta-refresh=SECONDS     How often to refresh
-  --output-stylesheet-link=URI      Link to an external CSS stylesheet
-  --output-title=TITLE              Page title
+Output Options:
+  --output-as=FORMAT                Specify output format as one of: console (default), html, text, xml
+  --output-to=DEST                  Specify file name for output (or "-" for stdout)
+  --html-cgi                        Add text needed to use output in a CGI program
+  --html-stylesheet-link=URI        Link to an external CSS stylesheet
+  --html-title=TITLE                Page title
+  --text-fancy                      Use more highly formatted output
```
Comment (kgaillot) [inline doc/Pacemaker_Administration/en-US/Ch-Tools.txt]: I do think it looks great like this. A tiny tweak, let's drop the "-link" for brevity

File: lib/common/cmdline.c [hunk 2]
```diff
@@ -89,8 +90,13 @@ pcmk__build_arg_context(pcmk__common_args_t *common_args, const char *fmts) {
             { NULL }
         };
 
-        output_main_entries[0].description = crm_strdup_printf("Specify output format as one of: %s", fmts);
-        g_option_context_add_main_entries(context, output_main_entries, NULL);
+        if (*output_group == NULL) {
+            *output_group = g_option_group_new("output", "Output Options:", "Show output help", NULL, NULL);
+        }
+
+        output_entries[0].description = crm_strdup_printf("Specify output format as one of: %s", fmts);
```
[the remark this answers is not in the document]
Comment (clumens) [reply lib/common/cmdline.c]: Ah, yeah, good point.  g_option_context_free is only called in three places now, all in tools that have been converted over.  I'm inclined to go ahead and add a wrapper now before it starts getting more widely used.

File: doc/Pacemaker_Administration/en-US/Ch-Tools.txt [hunk 3]
```diff
@@ -16,19 +16,21 @@ command line options for changing the style of the output.  For instance:
...
-Output Options (html):
-  --output-cgi                      Add text needed to use output in a CGI program
-  --output-meta-refresh=SECONDS     How often to refresh
-  --output-stylesheet-link=URI      Link to an external CSS stylesheet
-  --output-title=TITLE              Page title
+Output Options:
+  --output-as=FORMAT                Specify output format as one of: console (default), html, text, xml
+  --output-to=DEST                  Specify file name for output (or "-" for stdout)
+  --html-cgi                        Add text needed to use output in a CGI program
+  --html-stylesheet-link=URI        Link to an external CSS stylesheet
```
Comment (kgaillot) [inline doc/Pacemaker_Administration/en-US/Ch-Tools.txt]: missed the -link here

File: lib/common/cmdline.c [hunk 4]
```diff
@@ -94,7 +96,8 @@ pcmk__build_arg_context(pcmk__common_args_t *common_args, const char *fmts,
             *output_group = g_option_group_new("output", "Output Options:", "Show output help", NULL, NULL);
         }
 
-        output_entries[0].description = crm_strdup_printf("Specify output format as one of: %s", fmts);
+        output_as_descr = crm_strdup_printf("Specify output format as one of: %s", fmts);
+        output_entries[0].description = output_as_descr;
```
Comment (kgaillot) [inline lib/common/cmdline.c]: I'm thinking we should keep output_as_descr in common_args. Then free_common_args() can free it. That makes pcmk__free_arg_context() unnecessary, though we could keep it if we think more will end up there.

**Pull request merged**